docs/guidelines/releases-guidelines/07-api-gateway-release-process.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
Maybe we should change the number of this document from `07-api-gateway-release-process.md`
to `04-api-gateway-release-process.md`, because I think it makes sense that the release process docs are right next to each other.

Reply:
we should address this to the documentation team, I really don't have opinion on it.

Reply:
It is a good idea to renumber the docs if that's motivated by their logical relationship.